lib/temporal/connection.rb

```diff
@@ -10,12 +10,17 @@ def self.generate(configuration)
       connection_class = CLIENT_TYPES_MAP[configuration.type]
       host = configuration.host
       port = configuration.port
+      credentials = configuration.credentials || :this_channel_is_insecure
```

Comment:
nit: How about putting the default value into `Configuration` (along with other defaults)?